cli-plugins/insight/cli-overview.hbs.md

```diff
@@ -6,9 +6,16 @@ Follow the below steps to install, configure, and use the Tanzu Insight CLI plug
 
 >**Note:** Prior to using the CLI plug-in, you must install the Supply Chain Security Tools - Store, either as its own package, or as part of Tanzu Application Platform Build profile or Tanzu Application Platform View profile.
 
-1. [Install the Tanzu Insight CLI plug-in](cli-installation.md)
-1. [Configure target endpoint and certificate](../../scst-store/using-encryption-and-connection.md)
-1. [Configure access tokens](../../scst-store/create-service-account-access-token.md)
```

Comment:
These files are also linked to from the insight plug-in section of the TOC. So you should modify that section of the TOC.

Reply:
I've removed those from the toc